lib/threads.cpp

```diff
@@ -511,6 +541,22 @@ int GLVisCommand::Execute()
          break;
       }
 
+      case AXIS_NUMBERFORMAT:
+      {
+         cout << "Command: axis_numberformat: '" << axis_formatting << endl;
+         (*vs)->SetAxisNumberFormat(axis_formatting);
+         MyExpose();
+         break;
+      }
+
+      case COLORBAR_NUMBERFORMAT:
+      {
+         cout << "Command: colorbar_numberformat: '" << colorbar_formatting << endl;
```

Comment:
Missing closing apostrophe (and above too).

Reply:
Thanks